### ISSUE TITLE
PI(3,4,5)P3, PI(3,5)P2, and PI(3)P  phosphatase synonyms

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#15523]
PI(3,4,5)P3 phosphatase  found GO:0016314 | phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
add synonyms PtdIns(3,4,5)P3 phosphatase / PI(3,4,5)P3 phosphatase

PI(4,5)P2 phosphatase found  GO:0004439 | phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
add synonym  PI(4,5)P2 phosphatase

~PI(3,5)P2 phosphatase    can't find?~

~PI(3)P phosphatase  can't find?????~

**Comment by ValWood** [comment]
In the end I did not need the ones i could not find (not physiologically relevant for the gene I was annotating)....synonyms would  be useful though if they exist...

**Comment by krchristie** [comment]
Do you have specific suggestions for what synonyms you'd like to see on which terms?

**Comment by ValWood** [comment]
Just the ones above? (the abbreviations on the rows "add synonyms"). I was searching with these strings first and could not locate.

If you can find the 2 I can't find, the other abbreviations could be added too.

**Comment by krchristie** [comment]
@ValWood - If you have specific synonyms that you would like to have added to specific terms, please list them and I'll add them.

Otherwise, searching for lipid nomenclature that I'm not particularly well versed in is beyond the call of duty for me.

**Comment by krchristie** [comment]
These two synonyms seem fine for this term
> PI(3,4,5)P3 phosphatase found GO:0016314 | phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
> add synonyms PtdIns(3,4,5)P3 phosphatase / PI(3,4,5)P3 phosphatase

However, it seems odd to add PI(4,5)P_**_2_**_ as a synonym for a phosphatidylinositol-4,5-bisphosphate **_5_**-phosphatase activity, so could you confirm what you intended to request here:
> PI(4,5)P2 phosphatase found GO:0004439 | phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
> add synonym PI(4,5)P2 phosphatase

**Comment by ValWood** [comment]
GO:0004439    phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
is the term I used for  "PI(4,5)P2" which is what I was using to search, but did not retrieve it.
I found it eventually by a circuitous route....

I was assuming that it was the correct term, but this is only based on the fact that it has
PtdIns(4,5)P2 phosphatase activity as an exact synonym already.
So, if  this isn't GO:0004439 I used the incorrect thing?

**Comment by mah11** [comment]
> However, it seems odd to add PI(4,5)P_2_ as a synonym for a phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity, ...

Having looked at a few of these for one of Val's phenotype requests, I think the synonyms work such that the '(4,5)' indicates phosphorylated position(s), and the '2' simply indicates how many phosphate groups. If I've got it right, the suggested synonym makes some sense, although it looks broad rather than exact (the synonym only describes removing one phosphate from PI-4,5-bisphosphate; it doesn't specify which phosphate goes).

**Comment by ValWood** [comment]
I wasn't suggesting these synonyms because I needed them, I was suggesting them to help other curators to find the same terms *in future*, because I struggled to locate them. If this is not within the remit of an ontology editor, then feel free to close for me :)

I can reduce my  workload an not suggest cases synonyms could be added and leave  subsequent curators to  do the work ....

**Comment by ukemi** [comment]
I think it is fine for curators to request synonyms if they have literature support. In general these requests should be given the same considerations of any other term-related request. They should be backed up by a literature citation and the synonym scope should be specified.

**Comment by ValWood** [comment]
I forgot the PMID it's PMID:15249580

**Comment by ValWood** [comment]
It's odd that I could not find 
phosphatidylinositol-3-phosphatase
PI(3)P phosphatase
PIP3 phosphatase etc

with a search on *anything*.  Although I found the process term:
GO:0036092    phosphatidylinositol-3-phosphate biosynthetic process

?

**Comment by ValWood** [comment]
or
Phosphatidylinositol(3,4,5)-trisphosphatasase

(wikipedia calls the lipid phosphatidylinositol(3,4,5)-trisphosphate )
https://en.wikipedia.org/wiki/Phosphatidylinositol_(3,4,5)-trisphosphate
with
PtdIns(3,4,5)P3), abbreviated PIP3

so maybe there is no physiologically relevant phosphatase.

**Comment by ValWood** [comment]
OK this makes sense, and aligns with the paper I was reading:

" Subsequent studies demonstrated that in vivo the enzyme originally identified by Cantley's group prefers PtdIns(4,5)P2 as a substrate, producing the product PIP"

so only the first 2.

**Comment by ValWood** [comment]
thanks Alan,

OK  the general GO term for PI(3)P phosphatase
PtdIns(3,4,5)P3 phosphatases seems to be:
phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
https://www.ebi.ac.uk/QuickGO/term/GO:0016314

I don't know why I could not find it with my search strings, possibly to do with resolving hyphens and brackets, AmiGO and QuickGO do not cope well if you don't get them exact.....

**Comment by krchristie** [comment]
@ValWood - Since this thread has gone back and forth a bit, could you summarize the actions needed by listing the GO IDs with all of the synonyms that you would like added to each. Also please indicate the synonym type (EXACT, NARROW, BROAD, RELATED) and any references.

**Comment by ValWood** [comment]
GO:0016314  phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
add synonyms exact  PtdIns(3,4,5)P3 phosphatase / PI(3,4,5)P3 phosphatase

GO:0004439  phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
add synonym PI(4,5)P2 phosphatase (midori says broad)

GO:0016314 phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
add  synonym exact PI(3)P phosphatase

ref PMID:15249580

cheers,
val

**Comment by krchristie** [comment]
I should use that one PMID for the synonyms for all three of these GO terms?

**Comment by ValWood** [comment]
Yes all of these abbreviations come from this paper.

**Comment by mah11** [comment]
> GO:0016314 phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
> add synonyms exact PtdIns(3,4,5)P3 phosphatase / PI(3,4,5)P3 phosphatase
> 
> GO:0004439 phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
> add synonym PI(4,5)P2 phosphatase (midori says broad)

As far as I can tell, both of these synonyms should be broad, not exact, because the synonyms don't specify which phosphate is removed. They could be edited to be exact thusly:

GO:0016314 phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
exact synonyms PtdIns(3,4,5)P3 3-phosphatase / PI(3,4,5)P3 3-phosphatase
 
GO:0004439 phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
exact synonym PI(4,5)P2 5-phosphatase

**Comment by krchristie** [comment]
I want to check on some things.

1. @ValWood - I want to confirm that you only have two terms you want to add synonyms to. In summary of action items, you listed GO:0016314 two times for a total of three new synonyms. Is that correct, to add all three synonyms to this term:

id: GO:0016314
name: phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
namespace: molecular_function
def: "Catalysis of the reaction: phosphatidylinositol-3,4,5-trisphosphate + H2O = phosphatidylinositol-4,5-bisphosphate + phosphate." [EC:3.1.3.67]
synonym: "1-phosphatidyl-1D-myo-inositol-3,4,5-trisphosphate 3-phosphohydrolase activity" EXACT [EC:3.1.3.67]
synonym: "MMAC1" RELATED [EC:3.1.3.67]
synonym: "phosphatidylinositol-3,4,5-trisphosphate 3-phosphohydrolase activity" EXACT [EC:3.1.3.67]
NEW: **synonym: "PI(3)P phosphatase activity" EXACT [PMID:15249580]**
NEW: **synonym: "PI(3,4,5)P3 phosphatase activity" EXACT [PMID:15249580]**
NEW: **synonym: "PtdIns(3,4,5)P3 phosphatase activity" EXACT [PMID:15249580]**
xref: EC:3.1.3.67
xref: MetaCyc:3.1.3.67-RXN
xref: RHEA:25020
is_a: GO:0034594 ! phosphatidylinositol trisphosphate phosphatase activity


2. @mah11 - I am willing to defer to you on whether the synonyms should be BROAD or EXACT, but want to point out that some of the existing synonyms of GO:0004439 that seem to differ from the new ones only in punctuation are already designated as EXACT. I've highlighted the relevant new and existing synonyms below. Should I change them all to BROAD?

id: GO:0004439
name: phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
namespace: molecular_function
alt_id: GO:0001668
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol 4-phosphate **+** phosphate." [EC:3.1.3.36, RHEA:22767]
synonym: "phosphatidyl-myo-inositol-4,5-bisphosphate phosphatase activity" EXACT [EC:3.1.3.36]
synonym: "phosphatidylinositol 4,5-bisphosphate phosphatase activity" BROAD [EC:3.1.3.36]
synonym: "phosphatidylinositol-bisphosphatase activity" BROAD [EC:3.1.3.36]
NEW: **synonym: "PI(4,5)P2 phosphatase activity" EXACT [PMID:15249580]**
EXISTING: **synonym: "PtdIns(4,5)P(2) phosphatase activity" EXACT [EC:3.1.3.36]**
EXISTING: **synonym: "PtdIns(4,5)P2 phosphatase activity" EXACT [EC:3.1.3.36]**
synonym: "triphosphoinositide phosphatase activity" BROAD [EC:3.1.3.36]
synonym: "triphosphoinositide phosphomonoesterase activity" BROAD [EC:3.1.3.36]
xref: EC:3.1.3.36
xref: KEGG:R04404
xref: RHEA:22767
is_a: GO:0034595 ! phosphatidylinositol phosphate 5-phosphatase activity
is_a: GO:0106019 ! phosphatidylinositol-4,5-bisphosphate phosphatase activity

**Comment by mah11** [comment]
With a great big huge caveat that I'm not an expert on phosphoinositides or the kinases and phosphatases that act on them, yes, just from eyeballing the names and synonyms for the phosphoinositides I think several synonyms should be broad, including some of the ones currently labeled exact.

The difference between the EXACT and BROAD synonyms for the phosphatase terms lies in whether they do (exact) or don't (broad) specify the position that is dephosphorylated. If I understand the abbreviated substance synonyms correctly, the '2' or '3' at the ends of strings like PI(3,4,5)P3 only refers to the number of phosphate groups present, i.e. ending with 'P2' is equivalent to 'bisphosphate' and 'P3' is equivalent to 'trisphosphate'. It's the part in parentheses that specfies position(s).

So to specify which position a phosphatase dephosphorylates, there would have to be a prefix on the word 'phosphatase', as in some of the other synonyms. To illustrate, the synonym 'PI(3,4,5)P3 phosphatase' could refer to either of the reactions that Alan pointed out above:

PtdIns(3,4,5)P3 => PtdIns(4,5)P3 - see https://www.rhea-db.org/reaction?id=25017
PtdIns(3,4,5)P3 => PtdIns(3,4)P3 - see https://www.rhea-db.org/reaction?id=25529

An exact synonym for the first would be 'PI(3,4,5)P3 3-phosphatase', and for the second 'PI(3,4,5)P3 5-phosphatase'.

Summary for the GO terms you specifically listed (for brevity, showing only IDs, names, and synonyms):

id: GO:0016314
name: phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
synonym: "1-phosphatidyl-1D-myo-inositol-3,4,5-trisphosphate 3-phosphohydrolase activity" EXACT [EC:3.1.3.67]
synonym: "MMAC1" RELATED [EC:3.1.3.67]
synonym: "phosphatidylinositol-3,4,5-trisphosphate 3-phosphohydrolase activity" EXACT [EC:3.1.3.67]
NEW: synonym: "PI(3)P phosphatase activity" EXACT [PMID:15249580] **add as BROAD**
NEW: synonym: "PI(3,4,5)P3 phosphatase activity" EXACT [PMID:15249580] **add as BROAD**
NEW: synonym: "PtdIns(3,4,5)P3 phosphatase activity" EXACT [PMID:15249580] **add as BROAD**

**or** edit synonyms to
NEW: synonym: "PI(3)P **3-phosphatase** activity" EXACT [PMID:15249580]
NEW: synonym: "PI(3,4,5)P3 **3-phosphatase** activity" EXACT [PMID:15249580]
NEW: synonym: "PtdIns(3,4,5)P3 **3-phosphatase** activity" EXACT [PMID:15249580] 

(or even both, but that might be a bit much)

id: GO:0004439
name: phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
namespace: molecular_function
synonym: "phosphatidylinositol 4,5-bisphosphate phosphatase activity" BROAD [EC:3.1.3.36]
synonym: "phosphatidylinositol-bisphosphatase activity" BROAD [EC:3.1.3.36]
NEW: synonym: "PI(4,5)P2 phosphatase activity" EXACT [PMID:15249580] **add as BROAD**
EXISTING: synonym: "PtdIns(4,5)P(2) phosphatase activity" EXACT [EC:3.1.3.36]  **change to BROAD**
EXISTING: synonym: "PtdIns(4,5)P2 phosphatase activity" EXACT [EC:3.1.3.36] **change to BROAD**
synonym: "triphosphoinositide phosphatase activity" BROAD [EC:3.1.3.36]
synonym: "triphosphoinositide phosphomonoesterase activity" BROAD [EC:3.1.3.36]

**or**
NEW: synonym: "PI(4,5)P2 **5-phosphatase** activity" EXACT [PMID:15249580]
EXISTING: synonym: "PtdIns(4,5)P(2) **5-phosphatase** activity" EXACT [EC:3.1.3.36] 
EXISTING: synonym: "PtdIns(4,5)P2 **5-phosphatase** activity" EXACT [EC:3.1.3.36]

**Comment by ValWood** [comment]
Thanks Midori! I will refrain from requesting synonyms in future when I can't find terms by the search term I expected unless I know they  are equivalent!

**Comment by krchristie** [comment]
@mah11 - Thanks! for the very helpful comments on the synonyms. I agree with you that it might be a bit much to put in both the EXACT and the BROAD versions, so I have chosen the EXACT ones since I find them more helpful, being somewhat naive with my lipid nomenclature.
